Update SDK according to `2.0.0-pre-rc.14` (#153)
hyperledger/iroha-javascript: .changeset/large-teachers-boil.md

---
'@iroha2/data-model-schema': major
'@iroha2/data-model': major
'@iroha2/client': major
---

Updated according to Iroha `2.0.0-pre-rc.14` (TODO put finaly stable hash here)

Notable changes:

- Structure renamed: ~~`QueryError`~~ `QueryExecutionFailure`

> - `PublicKey` and `PrivateKey` had untyped `digest_function: string` field. Now the new enum struct is introduced - `Algorithm`:

Author: I would say:

> Introduced new enum struct, `Algorithm`. `digest_function: string` field in `PublicKey` and `PrivateKey` became `digest_function: Algorithm`:

> - `PublicKey` and `PrivateKey` had untyped `digest_function: string` field. Now the new enum struct is introduced - `Algorithm`:

Reviewer: then maybe

```suggestion
- Introduced new enum struct, `Algorithm`. Changed `digest_function: string` field in `PublicKey` and `PrivateKey` to `digest_function: Algorithm`:
```
